### PR TITLE
SYS-3828: updated avn-transaction-payment

### PR DESCRIPTION
**Pull request by ivan-cholakov**
Updated avn-transaction-payment pallet

### CODE REVIEW

File: pallets/avn-transaction-payment/Cargo.toml [hunk 1]
```diff
@@ -13,44 +13,48 @@ rust-version = { workspace = true }
...
+sp-runtime = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v1.0.0" }
+sp-application-crypto = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v1.0.0" }
+frame-support = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v1.0.0" }
+frame-system = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v1.0.0" }
+pallet-transaction-payment = { git = "https://github.com/paritytech/substrate", default-features = false, branch = "polkadot-v1.0.0" }
 
 # Optional imports for benchmarking
-serde = { version = "1.0.137", optional = true }
-frame-benchmarking = { default-features = false, git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.36", optional = true }
+# serde = { version = "1.0.163", default-features = false, optional = true}
```
Comment (thadouk) [inline pallets/avn-transaction-payment/Cargo.toml:31]:
```suggestion
```

File: pallets/avn-transaction-payment/Cargo.toml [hunk 2]
```diff
@@ -13,44 +13,48 @@ rust-version = { workspace = true }
...
+sp-io = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v1.0.0" }
+substrate-test-utils = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v1.0.0" }
+pallet-balances = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v1.0.0", features=["insecure_zero_ed"] }
 
 [features]
 default = ["std"]
 std = [
 	"frame-benchmarking?/std",
-	"serde",
+	# "serde/std",
```
Comment (thadouk) [inline pallets/avn-transaction-payment/Cargo.toml:43]:
```suggestion
```

**Review by thadouk — APPROVED** [review]

File: pallets/avn-transaction-payment/Cargo.toml [hunk 3]
```diff
@@ -13,44 +13,48 @@ rust-version = { workspace = true }
...
 	"codec/std",
+	"scale-info/std",
 	"sp-std/std",
+    "sp-core/std",
```
Comment (nahuseyoum) [inline pallets/avn-transaction-payment/Cargo.toml:47]:
```suggestion
	"sp-core/std",
```

File: pallets/avn-transaction-payment/src/tests/mock.rs [hunk 4]
```diff
@@ -1,4 +1,4 @@
-use crate::{self as pallet_avn_transaction_payment, system::limits, AvnCurrencyAdapter, Store};
+use crate::{self as pallet_avn_transaction_payment, system::limits, AvnCurrencyAdapter, KnownSenders};
```
Comment (nahuseyoum) [inline pallets/avn-transaction-payment/src/tests/mock.rs:1]: Do we need `KnownSenders`?

**Review by nahuseyoum — APPROVED** [review]

**Pull request merged**